docs/usage.rst

```diff
@@ -50,15 +50,15 @@ Tools
 tools.stainnorm
 ^^^^^^^^^^^^^^^
 .. automodule:: tiatoolbox.tools.stainnorm
-    :members: StainNormaliser, CustomNormaliser, RuifrokNormaliser, ReinhardNormaliser, get_stain_normaliser
+    :members: StainNormaliser, CustomNormaliser, RuifrokNormaliser, MacenkoNormaliser, VahadaneNormaliser, ReinhardNormaliser, get_stain_normaliser
```

Comment:
The last two must be `ReinhardColourNormaliser` and `get_normaliser`. I have fixed this in docs-fix-warnings branch but you can update it here.

Reply:
I will change so that we are now using `ReinhardNormaliser` throughout